mixer/pkg/runtime/dispatcher/dispatcher.go

```diff
@@ -115,6 +119,13 @@ const (
 	defaultValidUseCount = 10000
```

Comment:
I am wondering if increasing this to 100k or 1m would have the same effect without any other change ?

Reply:
My gut is that we aren't seeing great cache perf when no policies are in place because the key includes "source.uid" and "source.ip" among other attributes.

Testing is definitely needed though.